docs/web/tutorial-common-lifecycle-hooks.mdx

```diff
@@ -1,59 +1,63 @@
+Event gets executed when Web begins to initialize. Commands or workflows used here must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading. This can be useful for launching workflows that are going to impact the initial state of the application (such as initial layout).
 
-### Application Initialized
+## Application Initialized
 
-Event gets executed when Web has initialized. Commands or workflows must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading in Designer.
+This event gets executed when Web has initialized. Commands or workflows used here must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading in Designer.
 
 # Components > Map > Events >
 
-### Map Initialized
+## Map Initialized
 
-Event that gets triggered when the map is initialized. The map is initialized when it is loaded for the first time, a user signs in or out, or a map area is activated or deactivated. This can be useful for displaying a workflow that presents UI such as a form or alert or prompt.
+This is an event that gets triggered when the map is initialized. The map is initialized when it is loaded for the first time, a user signs in or out, or in Mobile when map area is activated or deactivated. This can be useful for displaying a workflow that presents UI such as a form or alert or prompt.
 
 # Services > Launch Link Workflows
 
-Using the Launch Link service, you can add and remove workflows to the app that can be optionally specified and run by URL parameter. If specified these workflows are run at the same time as the Map Initialized Event.
+Using the Launch Link service you can add and remove workflows to the app that can be optionally specified and run by URL parameter. The launch link workflow is run at the same time as the Map Initialized Event.
+
+<img
+    src={useBaseUrl("img/web-workflow-lifecycle-launch-link-3.png")}
+/>
 
 You can generate these URLs via Web Designers App Launch Link Dialog. (Deploy > Development > 3 Dot Menu > Get Links)
 
 <img
     src={useBaseUrl("img/web-workflow-lifecycle-launch-link-1.png")}
 />
 
+Use the generated link to run the specified workflow at startup.
+
 <img
     src={useBaseUrl("img/web-workflow-lifecycle-launch-link-2.png")}
 />
 
 # Event Service
 
-You can configure a workflow to run on any VertiGIS Studio Web event by registering an event listener in the Event service. [Events and Operations](https://developers.vertigisstudio.com/docs/web/api-commands-operations-events/#events)
+You can configure a workflow to run on any VertiGIS Studio Web event by registering an event listener in the Event service. [Events and Operations](https://developers.vertigisstudio.com/docs/web/api-commands-operations-events/#events) [Generic Event Listener](https://developers.vertigisstudio.com/blog/2023/03/01/mobile-generic-event-listener/)
```

Comment:
```suggestion
You can configure a workflow to run on any VertiGIS Studio Web event by registering an event listener in the Event service.
[Events and Operations documentation](https://developers.vertigisstudio.com/docs/web/api-commands-operations-events/#events)
[Generic Event Listener blog post](https://developers.vertigisstudio.com/blog/2023/03/01/mobile-generic-event-listener/)
```
Maybe just adding a bit more context for what the two links are, and putting them on separate lines makes it more obvious that there are two separate links rather than having them back-to-back.

Reply:
added the links inline